.gitignore

```diff
@@ -0,0 +1,4 @@
+.vscode
+Downloads
+json
```

Comment:
"json" is the name of the directory in the archive. I gitignore it to make it easy to just copy the whole thing in for testing.